submissions/YuraZagor/Frogger/app.js

```diff
@@ -0,0 +1,108 @@
+const allEnemies = [];
+const speedLow = 200;
+const speedMedium = 275;
+const speedHigh = 350;
+const allSpeeds = [speedLow, speedMedium, speedHigh];
+const enemyRestartX = -55;
+const collisionOffset = 40
+const cellX = 101;
+const cellY = 83;
+const spriteRepositioningY = -10
+const playerStartX = 2*cellX;
+const playerStartY = 5*cellY + spriteRepositioningY ;
+const fieldWidth = 5*cellX;
+const fieldHight = 5*cellY;
+const row1 = cellY + spriteRepositioningY
+const row2 = 2*cellY + spriteRepositioningY
+const row3 = 3*cellY + spriteRepositioningY
```

Comment:
Why mixed style of semicolons usage?

Reply:
My bad
Actually, I didn't have the habbit to use those before the  course,  and now I try to get used to them but at times I forget in haste